tests/sample.js

```diff
@@ -0,0 +1,199 @@
+/**
```

Comment:
Why is this file name sample.js? If this will be the only test file (small module testing), it should be called tests.js. If you will be testing multiple modules without the repo, it should probably be called sla.js.